src/coins.h

```diff
@@ -310,7 +310,7 @@ struct CCacheEntry
 typedef CCacheEntry<CCoins> CCoinsCacheEntry;
 typedef boost::unordered_map<uint256, CCoinsCacheEntry, SaltedTxidHasher> CCoinsMap;
 typedef std::map<uint256, CProposal> CProposalMap;
-typedef boost::unordered_map<uint256, CPaymentRequest, SaltedTxidHasher> CPaymentRequestMap;
+typedef std::map<uint256, CPaymentRequest> CPaymentRequestMap;
```

Comment:
https://en.cppreference.com/w/cpp/container/unordered_map

what i would say here, is that in `std::map` the order is guaranteed while `boost::unordered_map` does not guarantee it. guaranteed order is needed for cfunddbstate hash consistency. SaltedTxidHasher is removed because now we use `std::map`